src/lib/utils/statusFileManager.ts

```diff
@@ -77,11 +77,13 @@ export class StatusFileManager {
       if (!this.status.devhubs[devHubConfig.hubOrgAdminUsername].scratchOrgs)
         this.status.devhubs[devHubConfig.hubOrgAdminUsername].scratchOrgs = {};
       if (!this.status.devhubs[devHubConfig.hubOrgAdminUsername].scratchOrgs[devHubConfig.scratchOrgAdminUsername])
-        this.status.devhubs[devHubConfig.hubOrgAdminUsername].scratchOrgs[devHubConfig.scratchOrgAdminUsername] =
-          new ScratchOrg();
+        this.status.devhubs[devHubConfig.hubOrgAdminUsername].scratchOrgs[
```

Comment:
are these just formatting changes?

Reply:
These are lint --fix changes which Trina probably had to run to commit the changes. It is just formatting changes and the same changes exist in the develop branch as well (I did lint --fix before). This part is not significant to the PR.

Reply:
These were the old changes before lint --fix. I got it confused. I had to change to the old version again in the new PR.